components/todo-list/todo-list.component.html

```diff
@@ -1 +1,15 @@
-<p>todo-list works!</p>
+<main class="todo">
+    <div class="section-main">
+        <!-- <app-header></app-header> -->
+        <header class="header">
+            <h1>todos</h1>
+            <i class="fa-solid fa-chevron-down toggleButton" (click)="toggleAll()"></i>
+            <input type="text" class="new-todo" placeholder="What needs to be done?" [(ngModel)]="newTodo"
```

Comment:
This is ok. But lets talk about using here 'reactive forms'

Reply:
ok, I'll leave it for now.